src/HearThis/UI/DiscontiguousProgressTrackBar.cs

```diff
@@ -139,7 +139,7 @@ protected override void OnPaint(PaintEventArgs e)
 			//draw the bar
 			e.Graphics.FillRectangle(AppPallette.DisabledBrush, kLeftMargin, top, BarWidth, 3);
 
-			var brushes = GetSegmentBrushesOrErrorBrushes();
+			var brushes = GetSegmentBrushesMethod();
```

Comment:
Also, I can't actually find that method. GetSegmentBrushes(), below, appears to return a different length list of brushes depending on whether skipped ones are hidden, but nothing here seems to use a different number.